spec/models/delivery_company_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe DeliveryCompany, type: :model do
+  it "is valid with valid attributes" do
+    delivery_company = FactoryBot.create(:delivery_company)
+    expect(delivery_company).to be_valid
+  end
+
+  it "is valid without a name" do
+    delivery_company = FactoryBot.build(:delivery_company, name: nil)
+    expect(delivery_company).to be_valid
+  end
+
+  it "is valid without a phone number" do
+    delivery_company = FactoryBot.build(:delivery_company, phone_number: nil)
+    expect(delivery_company).to be_valid
+  end
```

Comment:
acá la idea sería tener solo un test para ver que la factory sea válida, [como esto](https://apidock.com/rails/ActiveRecord/ConnectionAdapters/SchemaStatements/add_reference#1665-foreign-key-option). Luego, en otro describe de validaciones, irían cosas como lo que tienes acá, pero hay un DSL particular para testear las validaciones comunes, [se ven así](https://github.com/platanus/the-store1/blob/master/spec/models/item_spec.rb#L8)